http_test.go

```diff
@@ -117,3 +117,82 @@ func TestLogin(t *testing.T) {
 		}
 	}
 }
+
+func TestNewUser(t *testing.T) {
+	k := Kasse{db: createDB(t), log: testLogger(t)}
+	k.sessions = sessions.NewCookieStore([]byte("TODO: Set up safer password"))
```

Comment:
It's completely acceptable to use "foobar" or something as the password here :) No need to confuse future-us with a TODO.